src/iedit.cpp

```diff
@@ -2368,8 +2412,78 @@ void iedit_parse(struct descriptor_data * d, const char *arg)
             case CYB_HANDBLADE:
             case CYB_HANDSPUR:
             case CYB_REFLEXTRIGGER:
-            case CYB_SKULL:
-            case CYB_TORSO:
+            case CYB_CYBERARMS:
+              if (number < 0 || number > 2) {
```

Comment:
It looks like there are 10 mods available, recommend `if (number < 0 || number > NUM_ARMS_MODS)` so that this is flexible in case future mods are added

Reply:
The same change should be made to the other inputs as well-- we want to remove magic numbers (numbers without explanation) wherever possible for better code readability.

Reply:
Good call, changed the reference for all of em to streamline any additional references down the line